.npmignore

```diff
@@ -2,11 +2,11 @@
 src
 babel.config.js
 .eslintrc
+.eslintignore
 .flowconfig
 .gitignore
 .gitmodules
-.travis.yml
-Makefile
```

Comment:
This deleted line probably belongs to the previous commit.

Reply:
indeed, moved to c82b3ffbba4fe08e700ea9ee75568c0895411351

Reply:
🖤